scss/common/_base.scss

```diff
@@ -387,6 +386,53 @@
         opacity: .3;
     }
 
+    $loading-size: 12px !default;
+    $loading-speed: 2s !default;
+    $loading-shadow-width: 2px !default;
```

Comment:
Is that the width of the animated stroke? If so, `$loading-stroke-width` may be more descriptive -- saying _what_ it is rather than _how_ it is implemented.